terraform-plans/modules/GitHub/templates/main.tf

```diff
@@ -61,7 +61,7 @@ resource "github_branch" "managed_files_branch" {
   count = length(local.changed_files) > 0 ? 1 : 0
 
   repository    = var.repository
-  branch        = "${var.pr_branch}-${random_string.update_uid.id}"
+  branch        = var.pr_branch
```

Comment:
You did not remove random_string (line 15-20).

Reply:
Thanks, updated.